http-core/src/main/scala/org/apache/pekko/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -336,13 +336,14 @@ private[http] object HttpServerBluePrint {
             openTimeouts = openTimeouts.tail
             push(responseOut, grab(responseIn))
           }
-          override def onUpstreamFinish() = complete(responseOut)
-          override def onUpstreamFailure(ex: Throwable) = fail(responseOut, ex)
+          override def onUpstreamFinish(): Unit = complete(responseOut)
+          override def onUpstreamFailure(ex: Throwable): Unit = fail(responseOut, ex)
         })
       setHandler(responseOut,
         new OutHandler {
-          def onPull(): Unit = pull(responseIn)
-          override def onDownstreamFinish() = cancel(responseIn)
+          def onPull():
+          Unit = pull(responseIn)
```

Comment:
Can you revert this change? It causes scalafmt check to fail.

Reply:
oops, I think I must have accidentally pressed enter here 😄

Reply:
Should be good now

Reply:
run with `scala-cli fmt`